deftree/svcparse/parser.go

```diff
@@ -336,9 +476,9 @@ func ParseHttpBindings(lex *SvcLexer) ([]*HTTPBinding, error) {
 	return nil, optErr
 }
 
-func ParseBindingFields(lex *SvcLexer) ([]*Field, error) {
-
+func ParseBindingFields(lex *SvcLexer) ([]*Field, []*Field, error) {
 	rv := make([]*Field, 0)
+	var custom []*Field
```

Comment:
Why is the constructed differently than `rv`?

```
rv := make([]*Field, 0)
```

Reply:
No reason at all, will update to make these both more consistent.